src/main/kotlin/no/nav/tilleggsstonader/sak/vilkår/vilkårperiode/domain/faktavurderinger/VurderingerMålgruppe.kt

```diff
@@ -10,29 +12,48 @@ sealed interface DekketAvAnnetRegelverkVurdering : VurderingerMålgruppe {
     val dekketAvAnnetRegelverk: VurderingDekketAvAnnetRegelverk
 }
 
+sealed interface AldersvilkårVurdering : VurderingerMålgruppe {
+    val aldersvilkår: VurderingAldersVilkår
```

Comment:
Hvordan er det tenkt at flyten skal være for denne? 
* Førstegangsbehandlinger når det opprettes
* Revurderinger som allerede har vilkårsvurderinger, men som mangler vurderingen fra før?

Reply:
Her tenker du på hvordan vi skal rulle dette ut i PROD, og samtidig hensynta eksisterende data?
Jeg snakket med Marie om dette, og vi kom frem til at vi nå begynner å lagre aldersvilkår, men prøver ikke å populere dette "bakover i tid".

Det innebærer i praksis at Aldersvilkår må være optional, da Vilkårsperioder som allerede finnes i databasen mangler aldersvurdering.

Alternativt så må man gjøre en migrering hvor alle eksisterende Vilkårsperioder(målgruppe) vurderes på nytt?

Reply:
Ok, då må den settes til nullable. Hvis ikke vil det feile for tidligere behandlinger?

Reply:
Siden `fakta_og_vurdering` kolonnen er JSON så vil det kanskje gå fint om tidligere behandlinger ikke inneholder `Aldersvilkår`? Så vidt jeg vet så valideres ikke innholdet i `fakta_og_vurdering` etter at vilkårsperioden er lagret?